packages/core/src/generators/imports.ts

```diff
@@ -47,7 +47,7 @@ export const generateImports = ({
         } } from \'./${upath.join(path, camel(name))}\';`;
       }
 
-      return `import ${!values ? 'type ' : ''}{ ${name}${
+      return `import { create${name}, ${name}${
```

Comment:
Since `create` exists in the http status, I want to change the function name to the `new` prefix to avoid confusion.

```suggestion
      return `import { new${name}, ${name}${
```